lib/angular_recaptcha.dart

```diff
@@ -136,6 +138,11 @@ class AngularRecaptcha extends ValueAccessor
     _ngModel?.viewToModelUpdate(v);
   }
 
+  /// Depending on the value, it will enable or disable the
+  /// appropriate DOM element.
+  @override
+  void onDisabledChanged(bool isDisabled) {}
```

Comment:
can you explain this ?

Reply:
It doesn't do anything, it is needed to be implemented by abstract class otherwise analyzer plugin will show an error.

This was implemented just to hide analyzer plugin errors.

Reply:
I copied documentation of abstract class.